.github/workflows/benchmarks.yml

```diff
@@ -0,0 +1,43 @@
+name: Run benchmarks
+on:
+    schedule:
+      # Run every day at midnight.
+      - cron: "0 0 * * *"
+
+jobs:
+  benchmark:
+    if: ${{ github.actor != 'dependabot[bot]' }}
+    name: run benchmarks
+    runs-on: ubuntu-latest
+    timeout-minutes: 120
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          repository: boa-dev/boa
+          path: boa
+      - uses: actions-rs/toolchain@v1.0.7
+        with:
+          toolchain: stable
+          override: true
+          profile: minimal
```

Comment:
Would be better to use dtolnay/rust-toolchain, since this action is unmaintained.